### DISCUSSION TITLE
Non-sticky group headers?

### DISCUSSION THREAD

**Discussion by going-confetti** [petyosi/react-virtuoso#586] [Q&A]
First of all, fantastic work on the library!

I have a question about the group headers in `GroupedVirtuoso`: is it possible to make them non-sticky? In my case, some of them are multiline, which means the previous group header could be shown even if all the items on the screen belong to the next group ([Sandbox](https://codesandbox.io/s/groupedvirtuoso-multiline-headers-uqxdjc)).

I tried a solution similar to what's described in https://github.com/petyosi/react-virtuoso/issues/73, but there's still one problem: once you've scrolled to the point where the group header would normally "stick", it disappears ([Sandbox](https://codesandbox.io/s/groupedvirtuoso-static-headers-ef6bt0?file=/App.js) - I'm assuming its content is then transferred to `TopItemList`).

I also found https://github.com/petyosi/react-virtuoso/issues/146 and the recommendation at that point was to just use flat list, but I'd prefer using the `GroupedVirtuoso` API if possible.

If there's still no option at the moment and you think it'd make sense to have one, I'd be happy to contribute.

**Comment by thiagoissao** [comment]
Hi @going-confetti, did you solve the non-sticky header in the group mode? I'm with the same problem here

**Reply by going-confetti** [reply]
Hi! No, unfortunately, I didn't find a good solution, so I resorted to overriding `position` for both `Group` and `TopHeader` components ([like this](https://codesandbox.io/s/groupedvirtuoso-static-headers-ef6bt0?file=/App.js)). it's far from perfect, because now headers disappear instead of sticking, but in my case it was acceptable.
